app/controllers/UserController.php

```diff
@@ -30,6 +30,8 @@ public function __construct()
         $tagRepository = new TagRepository($dbConnection);
         $difficultyRepository = new DifficultyRepository($dbConnection);
         $unitRepository = new UnitRepository($dbConnection);  // Instantiate UnitRepository
+        $likeRepository = new LikeRepository($dbConnection);  // Add this line if it was missing
+
```

Comment:
I've noticed that we are using $dbConnection in some controllers, and $db in other, Maybe we should have either one through out the project?